test/test-resolve-hook.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<link rel="stylesheet" type="text/css" href="../node_modules/mocha/mocha.css"/>
+<script src="../node_modules/mocha/mocha.js"></script>
+<script type="importmap-shim">
+  {
+    "imports": {
+      "test": "/test/fixtures/es-modules/es6-file.js"
+    }
+  }
+</script>
+<script type="module-shim">
+  import test from "test";
+  console.log(test);
+</script>
+<script type="module">
+  mocha.setup('tdd');
+  mocha.allowUncaught();
+
+  self.baseURL = location.href.substr(0, location.href.lastIndexOf('/') + 1);
+  self.rootURL = location.href.substr(0, location.href.length - 'test/test.html'.length);
+  self.assert = function (val) {
+    equal(!!val, true);
+  };
+  assert.equal = equal;
+  assert.ok = assert;
+  function equal (a, b) {
+    if (a !== b)
+      throw new Error('Expected "' + a + '" to be "' + b + '"');
+  }
+  self.fail = function (msg) {
+    throw new Error(msg);
+  };
+
+  const suites = ['resolve-hook'];
```

Comment:
Rather than having a separate test-resolve-hook, can we just add this to the main suite in test/test.html? That will ensure it's captured by CI.